contracts/acl/ACL.sol

```diff
@@ -145,6 +150,20 @@ contract ACL is IACL, TimeHelpers, AragonApp, ACLHelpers {
         _setPermission(_entity, _app, _role, NO_PERMISSION);
     }
 
+    /**
+    * @dev Revokes all permissions for a specified role if allowed. This requires `msg.sender` to be the the permission manager
+    * @notice Revoke from all the ability to perform actions requiring `_role` on `_app`
+    * @param _app Address of the app in which the role will be revoked
+    * @param _role Identifier for the group of actions in app being revoked
+    */
+    function revokeAll(address _app, bytes32 _role)
+        external
+        onlyPermissionManager(_app, _role)
+    {
+        bytes32 hash = roleHash(_app, _role);
+        roleEras[hash] = roleEras[hash].add(1);
```

Comment:
Just realized, we should definitely emit an event here as otherwise it'd be very difficult to know this happened in a frontend.

Reply:
Sorry for the delay! I added a `RevokeAllPermissions` event :)